### DISCUSSION TITLE
The money transfer did not come to the wallet

### DISCUSSION THREAD

**Discussion by rimlyanin** [Chia-Network/chia-blockchain#9537] [Support]
Hallo
I created a Wallet Chia and transferred coins from the Huobi exchange to it. From the exchange left and on the wallet did not receive What to do? translated on the xch network  
xch1tpmpa3hvhkmcyupxugf4ff6td729vnr0fs75j80jezj7nvkx6q0sy7qzhp

**Comment by Jazz-Bass** [comment]
Same problem. I created a wallet Chia Blockchain and transferred coins from the gate.io. A day has passed, but the money was not received to the chia blockchain wallet. Hash:
0x18a2f957ea03ceb9ba794ee7d53a9bbf094d294e67215bf19c921f075ccbabe8
Please help admin!
